delta/service/src/main/scala/ch/epfl/bluebrain/nexus/delta/service/resources/ResourcesImpl.scala

```diff
@@ -30,123 +31,137 @@ final class ResourcesImpl private (
     projects: Projects,
     eventLog: EventLog[Envelope[ResourceEvent]]
 )(implicit rcr: RemoteContextResolution, uuidF: UUIDF)
-    extends Resources {
+    extends ResourcesCommons(projects)
```

Comment:
It is either having this class or just copy pasting all the methods in both `ResourcesDummy` or `ResourcesImpl`. Both options are ok for me

Reply:
I'm not fond of an abstract class either; I would also prefer an object where the common functionality can be defined; it can even be the companion object of Resources with private[delta].

Reply:
It is not an object, since it needs the `projects: Projects` and all this implicit parameters. So just having random methods in `Resources` companion object which do nothing there is not really useful, nor clear either.

I will rather duplicate all those methods then

Reply:
projects could stays in ResourcesImpl and the other stuff go to the object no ?